### ISSUE TITLE
Provide a means to contact other users

### ISSUE THREAD

**Issue by rudolfmayer** [mapillary/mapillary_issues#807]
There should be some kind of possibility to get in touch with other users. That might be a messaging system within the website, or a way to contact users via their email (e.g. also from a form at the website, and the email gets sent from the web application itself, thus not revealing any user email address).

At least at the moment, the only option to interact with other users seems to be via comments to pictures, but that is obviously not a preferred means...

**Comment by filipca** [comment]
Facebook could be a solution, but the only M user I know in Belgium has no Facebook account.
I made a Belgian Mozilla Location Services Facebook group, but I still have to congratulate myself.

**Comment by polarbearing** [comment]
I would not want to be forced to create a FB account to use mapillary.

**Comment by rudolfmayer** [comment]
I don't think that Facebook or other external solutions should be built in.

To clarify maybe: I was thinking more of a messaging service as it is e.g. built in in osm.org. There you basically have an email system, where you can read & write messages. You also receive a copy of inbound messages to your email address registered with your OSM account.

For mapillary, it can be even simpler than this. 
For example, the mapillary website only provides you a contact form (once logged in), and it sends an email to the user, and from there on, any subsequent communication is done via emails and external to mapillary (just set the reply-to header for the email the first user gets to the sender, and that should be it).

**Comment by jesolem** [comment]
I think @-mentions is the way to go now that we have that.

**Comment by rudolfmayer** [comment]
It seems that the @-mentions are limited to comments in photos as well, right?

One downside with the mentions is that it gets totally lost in the activity feed, if you are a frequent contributor (especially if you have many sequences). If you can't filter the activity by comments/mentions vs. e.g. upload activities, this will be hardly usable.

**Comment by gyllen** [comment]
@-mention should be filtered on new profile page, would also be cool to investigate means on how users can communicate with each others.

**Comment by malenki** [comment]
I'd need the functionality right now – want to ask a user about photos from a certain area…

**Comment by de-vries** [comment]
@gyllen, how about a comment section on the profile page to create a public message board or "wall" were everyone can post and read messages. 
Personal messages would also be nice, but public messages create a bigger sense of community I think.

**Comment by gyllen** [comment]
@de-vries Yes agree, we are thinking about handling this. Going to keep this ticket open for discussion.

**Comment by mrAceT** [comment]
Thought it would be there, couldn't find it.. So that is correct.. because it isn't there ;)

Would appreciate the option :)
